lib/util/call-taker.js

```diff
@@ -1,9 +1,66 @@
+import {isTransit} from '@opentripplanner/core-utils/lib/itinerary'
+import {randId} from '@opentripplanner/core-utils/lib/storage'
+
 import {getRoutingParams} from '../actions/api'
+import {getTimestamp} from './state'
+
+export const TICKET_TYPES = {
+  own_tickets: 'Will use own tickets',
+  hop_new: 'Will purchase new Hop Card',
+  hop_reload: 'Will reload existing Hop Card'
+}
+
+const PAYMENT_PREFS = {
+  request_call: 'Call requested at provided phone number',
+  phone_cc: 'Will call in credit card info to TriMet',
+  fax_cc: 'Will fax credit card info to TriMet',
+  mail_check: 'Will mail check to TriMet'
+}
```

Comment:
These are all TriMet-specific. These should be loaded from config instead.

Reply:
I don't think we need to worry about making the TriMet field trip module generic at this stage.

Reply:
Make an issue or point to some kind of roadmap item so this doesn't get lost. Add a comment in the code that technical debt is being added with a plan to pay it off in the future.

Reply:
Issue is here: https://github.com/opentripplanner/otp-react-redux/issues/353